Doc/Manual/Typemaps.html

```diff
@@ -734,7 +734,7 @@ <H3><a name="Typemaps_defining">12.2.1 Defining a typemap</a></H3>
 </p>
 
 <p>
-<em>modifiers</em> is an optional comma separated list of <tt>name="value"</tt> values. 
+<em>modifiers</em> is an optional comma separated list of <tt>name="value"</tt> or <tt>name={value}</tt> values. 
```

Comment:
It seems the hblock style (e.g. `name=%{value with a " in it%}`) is also now supported, which seems reasonable for consistency with typemap bodies, but it ought to be documented here too.

Reply:
We should probably also say "since SWIG 4.1.0" or whatever for new features like this.